tests/custom_hypothesis_support.py

```diff
@@ -0,0 +1,67 @@
+import astroid
+import hypothesis.strategies as hs
+from typing import Any, Dict, List, Tuple
+
+# Custom strategies for hypothesis testing framework
+primitive_types = hs.sampled_from([
+    hs.integers,
+    hs.booleans,
+    lambda: hs.floats(allow_nan=False, allow_infinity=False),
+    hs.none,
+    hs.text,
+])
+primitive_values = primitive_types.flatmap(lambda s: s())
+
+# Strategies for generating Indexes
+index_types = hs.sampled_from([
+    hs.integers,
+    lambda: hs.text(alphabet="abcdefghijklmnopqrstuvwxyz", min_size=1)
+])
+index_values = index_types.flatmap(lambda s: s())
+
+# strategy for generating integers
+integer = hs.integers()
```

Comment:
I don't think it's worth creating aliases for existing hypothesis strategies. You should remove this and just use the original strategy. SImilar for `string` below.